src/bakery/Client.h

```diff
@@ -5,9 +5,31 @@
 #ifndef FEUP_PROJETO_CAL_CLIENT_H
 #define FEUP_PROJETO_CAL_CLIENT_H
 
+#include <string>
+#include "../graph/Vertex.h"
+#include "../utils/Time.h"
 
+using namespace std;
+
+template<class T>
 class Client {
+private:
+    int id;
+    string name;
+    Vertex<T> address;
```

Comment:
Isn't the Vertex class already going to store the Client inside it?

Reply:
Okay from what I understood we have clients that store a vertex inside, which have a Position. But wouldn't it be easier to just have Vertices that contain a Client?